src/raft_node.c

```diff
@@ -128,7 +128,7 @@ void raft_node_clear_flags(raft_node_t *node)
     node->flags = 0;
 }
 
-void raft_node_vote_for_me(raft_node_t *node, const int vote)
+void raft_node_set_vote_for_me(raft_node_t *node, const int vote)
```

Comment:
```suggestion
void raft_node_set_voted_for_me(raft_node_t *node, const int vote)
```

Maybe we should match flag's name by adding that `d` : `RAFT_NODE_VOTED_FOR_ME`

Reply:
👍